javascript/javascript3/week3/circle.js

```diff
@@ -0,0 +1,94 @@
+
+
+function canvas()
+{
+const x=document.getElementById('myCanvas')
+const ctx=x.getContext("2d")
+
+// create N by lines
+ctx.beginPath();
+ctx.moveTo(100,100);
+ctx.lineTo(100, 200);
+ctx.stroke();
+
+ctx.beginPath();
+ctx.moveTo(200,100);
+ctx.lineTo(200, 200);
+ctx.stroke();
+ctx.beginPath();
+ctx.moveTo(100,100);
+ctx.lineTo(200, 200);
+ctx.stroke();
+ 
+// cirles with diffrent sizes
+for(let x=5 ,color=0,radius=10;x<10;x++)
+{
+
+    for(let y=5;y<10;y++)
+    {
+    ctx.beginPath();
+    ctx.arc(60*x, 60*y, radius, 0,2*Math.PI)
+    ctx.closePath();
+    ctx.strokeStyle= 'hsl('+color+',100%,50%)'
+    ctx.stroke();
+    radius+=1;
+    color+=17;
+ 
+    }
+}
```

Comment:
did you consider the following
1- make a function to drow circle? Because of single responsibility for each function.
2- starting for from 1, not 5
3- Why nested for? those for running 25 times
4- it is not recommended to assign color and radius on the for.